src/app/hierarchical-grid/hierarchical-grid-validator-service-extended/hierarchical-grid-validator-service-extended.component.html

```diff
@@ -1,23 +1,32 @@
 <div class="grid-wrapper">
-    <igx-hierarchical-grid #hierarchicalGrid height="570px" width="100%" [primaryKey]="'CustomerID'"
+    <igx-hierarchical-grid #hierarchicalGrid height="570px" width="100%" [primaryKey]="'CustomerID'" (formGroupCreated)="formCreateHandler($event)"
         [batchEditing]="true" [rowEditable]="true">
         <igx-column field="CustomerID" [editable]="false"></igx-column>
         <igx-column field="CompanyName" [editable]="true" required></igx-column>
-        <igx-column field="ContactName" [editable]="true" forbiddenName="bob" required>
+        <igx-column field="ContactName" [editable]="true" required>
         </igx-column>
         <igx-column field="ContactTitle" [editable]="true" required></igx-column>
-        <igx-column field="Phone" [editable]="true" required></igx-column>
+        <igx-column field="Phone" [editable]="true" required phoneFormat="\(?\d{3}\)?-? *\d{3}-? *-?\d{4}">
+            <ng-template igxCellValidationError let-cell='cell' let-defaultErr='defaultErrorTemplate'>
+                <ng-container *ngTemplateOutlet="defaultErr" >
+                    This Field is required
```

Comment:
Content here does not get projected in the default templates so it can be removed.